app/Http/Requests/Staff/UpdateGroupRequest.php

```diff
@@ -42,151 +42,175 @@ public function rules(Request $request): array
         $group = $request->route('group');
 
         return [
-            'name' => [
+            'group.name' => [
                 Rule::when(! $group->system_required, [
                     'required',
                     'string',
                 ]),
-                Rule::prohibitedIf($group->system_required && $request->name !== $group->name),
+                Rule::prohibitedIf($group->system_required && $request->group->name !== $group->name),
```

Comment:
This rule gives a 500 error when attempting to update a system required group, because `$request->group` is an array. Correct syntax would be `$request->group['name']`.

Reply:
Thanks for pointing this out. I remember fixing this exact error, but not sure where my fix went...